carma_cooperative_perception/cmake/FindPROJ.cmake

```diff
@@ -0,0 +1,91 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying
+# file COPYING-CMAKE-SCRIPTS or https://cmake.org/licensing for details.
+
+#[=======================================================================[.rst:
+FindPROJ
```

Comment:
Do we need this since we added the `FindPROJ` in cmake?

Reply:
Yeah it looks like FindPROJ.cmake is not actually needed so I removed it. The rest of the cmake folder is still in use though